src/det.jl

```diff
@@ -0,0 +1,8 @@
+function Base.LinAlg.det(M::Matrix{<:MultivariatePolynomials.AbstractPolynomialLike})
```

Comment:
This doesn't work in 0.7: `Base.LinAlg.det`. Maybe import `det` (see comment above)